forgerock-ui-commons/pom.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- ~ DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
      ~
-     ~ Copyright (c) 2011-2013 ForgeRock AS. All Rights Reserved
+     ~ Copyright (c) 2011-2013 ForgeRock AS. All Rights Reserved.
```

Comment:
Don't know if we need to touch this. ForgeRock is really sloppy with their copyright notices. For example [here](https://github.com/WrenSecurity/wrensec-parent/blob/fe797dddcd184514ed2e7c9c679b3113ca375ca7/pom.xml) they use `Copyright 2011-2015 ForgeRock AS.`. So we might just be better of not touching theirs.

Reply:
you're being too picky. but I'll change it back.